dcpy/test/connectors/edm/conftest.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from pathlib import Path
+import uuid  # for generating a unique temp directory name
+import os
+import shutil
+import csv
+from dcpy.connectors.edm import publishing
+
+
+@pytest.fixture(scope="module")
+def mock_data_constants():
+    constants = {
+        "TEST_DATA_DIR": Path(__file__).resolve().parent / str(uuid.uuid4()),
```

Comment:
Just curious, why do we want this randomized?

Reply:
@alexrichey, I wanted to make sure that there are no existing directories with the same name... Which is not a problem now, but the package could potentially evolve in the future and have new permanent directories. And since it's a temp directory that's only created for the duration of tests, the name shouldn't play a big role. What do you think?

Reply:
I'd probably use a non-random dir name, and do some teardown (ie `rm -rf`) at the beginning of the fixture. Maybe just to avoid accumulation of randomized directories in `edm` folder. (e.g. every now and then I'll do a ctrl-c in the middle of a test run... I'm not sure how gracefully pytest deals with that.) Again, not a blocker here 🙂

Reply:
Makes sense! I added your rec to my code.

By the way, I tried interrupting the tests with ctrl-c and it seems that pytest still does the teardown! Pleasant surprise :) 

<img width="773" alt="image" src="https://github.com/NYCPlanning/data-engineering/assets/144725249/c57c5e5a-2b4c-41b4-b685-64554a120ded">